envi/archs/arm/disasm.py

```diff
@@ -1690,10 +1748,11 @@ def repr(self, op):
 class ArmRegShiftRegOper(ArmOperand):
     ''' register shift operand.  see "addressing mode 1 - data processing operands - * shift * by register" '''
 
-    def __init__(self, reg, shtype, shreg):
+    def __init__(self, reg, shtype, shreg, display_shift = True):
```

Comment:
i'm not convinced this is necessary, and it definitely adds to complexity and maintenance-mess.